src/tests/system/tests/test_sss_override.py

```diff
@@ -77,6 +77,40 @@ def test_sss_overrides__overriding_username_and_posix_attributes(client: Client,
     assert result.home == "/home/o-user1", "User's override name homedir does not match override value!"
 
 
+@pytest.mark.importance("medium")
```

Comment:
This is similar to ```test_sss_overrides__overriding_group_name_and_gid``` and ```test_sss_overrides__overriding_username_and_posix_attributes```

I don't know that we need this test.

Reply:
Did I miss a test that makes sure that `getent group` show overridden names?

Reply:
This is important to test this PR.

Reply:
Ok, I will drop my test after review.

Reply:
Linking PR to update the test case
https://github.com/SSSD/sssd/pull/7887